### PR TITLE
Adds a pref for hearing instrument sounds

### PR DESCRIPTION
**Pull request by mikomyazaki**
Suggested here https://forums.aurorastation.org/topic/14618-please-for-the-love-of-god-add-a-preference-toggle-for-musical-instrument-sounds/?_fromLogin=1

Did I miss any instruments?

### CODE REVIEW

**Comment by mikomyazaki** [comment]
!review

File: code/__defines/misc.dm [hunk 1]
```diff
@@ -74,8 +74,9 @@
 #define ASFX_DROPSOUND	16
 #define ASFX_ARCADE		32
 #define ASFX_RADIO		64
+#define ASFX_INSTRUMENT 128
```
Comment (alsoandanswer) [inline code/__defines/misc.dm:77]: the spacings a little weird here

Comment (mikomyazaki) [reply code/__defines/misc.dm:77]: spacing

Not sure what's going on there... I blame git? :S

**Comment by alsoandanswer** [comment]
Wonder if you could move jukeboxes to the instrument toggle instead of the ambience one

**Comment by mikomyazaki** [comment]
> Wonder if you could move jukeboxes to the instrument toggle instead of the ambience one

Would need to change quite a bit, since Jukeboxes set an area's ambience rather than playing the sound directly at hearers.

File: SQL/migrate/V053_preference_asfx_instrument.sql [hunk 2]
```diff
@@ -0,0 +1,6 @@
+--
+-- Set default 'On' for 'asfx_instruments'
+-- 
+
+ALTER TABLE `ss13_characters`
+	ALTER 'asfx_togs' SET DEFAULT '128';
```
Comment (skull132) [inline SQL/migrate/V053_preference_asfx_instrument.sql:6]: This wasn't exactly what I had in mind. Or rather, this doesn't update the existing entries.

```suggestion
UPDATE `ss13_characters` SET `asfx_togs` = `asfx_togs` | 128;
```

Comment (skull132) [reply SQL/migrate/V053_preference_asfx_instrument.sql:6]: Also, the file name needs two underscores between the version number and the rest. so `V053__preference_asfx_intr...`

File: SQL/migrate/V053__preference_asfx_instrument.sql [hunk 3]
```diff
@@ -0,0 +1,7 @@
+--
+-- Set default 'On' for 'asfx_instruments'
+-- 
+
+UPDATE `ss13_player_preferences` SET `asfx_togs` = `asfx_togs` | 128;
+ALTER TABLE `ss13_player_preferences`
+	ALTER `asfx_togs` SET DEFAULT '128';
```
Comment (Arrow768) [inline SQL/migrate/V053__preference_asfx_instrument.sql:7]: I am not sure if that default value is correct.

Comment (skull132) [reply SQL/migrate/V053__preference_asfx_instrument.sql:7]:
```suggestion
	ALTER `asfx_togs` SET DEFAULT '255';
```

Comment (mikomyazaki) [reply SQL/migrate/V053__preference_asfx_instrument.sql]: Corrected this following discussion with Skull here <https://discordapp.com/channels/131477462461841409/131489773268238336/736665658875445328> by removing the ALTER line, as the default is set in-code already in /datum/preferences

**Pull request merged**